hc/api/models.py

```diff
@@ -195,7 +205,10 @@ def notify(self, check):
 
         if error != "no-op":
             n = Notification(owner=check, channel=self)
-            n.check_status = check.status
+            if check.status.lower() == "up" and check.ping_is_early:
```

Comment:
been looking into that Lewis. I am writing my tests. will be done hopefully by end of day tomorrow